releases/tickets/v23_0_0.rst

```diff
@@ -34,40 +66,37 @@ Tickets Addressed in Release 23.0.0
+- `DM-32198 <https://jira.lsstcorp.org/browse/DM-32198>`_:  Set sentinel values for flag/integer columns for missing bands in Object tables [pipe_tasks]
+- `DM-32201 <https://jira.lsstcorp.org/browse/DM-32201>`_:  HTCondor plugin bug if output collection contains period [ctrl_bps]
+- `DM-32217 <https://jira.lsstcorp.org/browse/DM-32217>`_:  BPS displays cumulative timings instead of time intervals during a submission [ctrl_bps]
+- `DM-32220 <https://jira.lsstcorp.org/browse/DM-32220>`_:  HTCondor plugin copying execution butler back from job [ctrl_bps]
+- `DM-32238 <https://jira.lsstcorp.org/browse/DM-32238>`_:  Fix unexpected floating point values in drpAssociation task [pipe_tasks]
+- `DM-32241 <https://jira.lsstcorp.org/browse/DM-32241>`_:  bps submit can take multiple hours [ctrl_bps]
+- `DM-32290 <https://jira.lsstcorp.org/browse/DM-32290>`_:  A create-table failure led to registry corruption  [daf_butler]
+- `DM-32305 <https://jira.lsstcorp.org/browse/DM-32305>`_:  lsst.daf.butler.Config slowness with large numbers of lookups [daf_butler]
+- `DM-32306 <https://jira.lsstcorp.org/browse/DM-32306>`_:  refBand functor doesn't know that u-band exists [pipe_tasks]
+- `DM-32333 <https://jira.lsstcorp.org/browse/DM-32333>`_:  SourceTable should use LocalNanojansky functor on apFluxes [obs_lsst, obs_subaru]
+- `DM-32435 <https://jira.lsstcorp.org/browse/DM-32435>`_:  Silent failure in executing memory-hungry job? [ctrl_bps]
+- `DM-32448 <https://jira.lsstcorp.org/browse/DM-32448>`_:  Backport incorrect GAaP field name [obs_lsst]
+- `DM-32504 <https://jira.lsstcorp.org/browse/DM-32504>`_:  SingleQuantumExecutor should log exception messages [ctrl_mpexec]
+- `DM-32594 <https://jira.lsstcorp.org/browse/DM-32594>`_:  Remove dependency on inflection in bps_config [ctrl_bps]
+- `DM-32624 <https://jira.lsstcorp.org/browse/DM-32624>`_:  Override source selection criteria in SFM calibration stages for LSSTCam-imSim [meas_algorithms]
+- `DM-32649 <https://jira.lsstcorp.org/browse/DM-32649>`_:  Move psf quality statistics calculation from selectImages to computeExposureSummaryStats [afw, pipe_tasks]
+- `DM-32714 <https://jira.lsstcorp.org/browse/DM-32714>`_:  Fix backwards compatibility broken by DM-32649 [pipe_tasks]
 
 Tickets merged before w_2021_40
 -------------------------------
 
-- `DM-6056 <https://jira.lsstcorp.org/browse/DM-6056>`_: Remove PTR and CONST_PTR macros [afw, base, daf_persistence, ip_diffim, meas_algorithms, meas_astrom, meas_base, meas_deblender, meas_extensions_photometryKron, meas_extensions_psfex, meas_modelfit, obs_subaru, shapelet]
+- `DM-6056 <https://jira.lsstcorp.org/browse/DM-6056>`_:  Remove PTR and CONST_PTR macros [afw, base, ip_diffim, daf_persistence, meas_algorithms, meas_astrom, meas_base, meas_deblender, meas_extensions_photometryKron, meas_extensions_psfex, meas_modelfit, obs_subaru, shapelet]
```

Comment:
You may want to update your generator to trim leading/trailing whitespace.

Reply:
Fixed